client/src/components/content/CharacterList.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState,useEffect } from "react"
+import CharacterTile from "./CharacterTile"
+
+const CharacterList = (props) => {
+    const [characters, setCharacters] = useState([])
```

Comment:
You'll want to sync with your group on tab size indentation - this file has 4 tab size, while other files have 2. You can change this setting in VSCode so there is consistency between all team members in your project!